### ISSUE TITLE
Remove polyfill.io script

### ISSUE THREAD

**Issue by brichwin** [NSoiffer/MathCATDemo#7]
Per email from Davide Cervone on the mathjax-dev google groups list (Wednesday, June 26, 2024 at 8:01 AM):

```
Folks:
 
In the past, MathJax has recommended including a <script> tag that refers to polyfill.io in order to include support for IE11 and other older browsers.  You should remove these script files from your websites.  The polydill.io site has been purchased by a Chinese company that has used it to inject malware into sites that use it, as described in this article:
 
https://sansec.io/research/polyfill-supply-chain-attack
 
There is really no need to include this polyfill any longer, as all modern browsers can run MathJax without it.
 
Please be sure to update your websites to remove this significant security issue!
 
Davide P. Cervone
MathJax lead developer
```

**Comment by NSoiffer** [comment]
Thanks for letting me know. I've spent the last few days trying to rebuild the project. For NVDA (and others), I changed the way files are handled (each language is zipped up to save space and unzipped upon usage). The web build handles files differently and I have been working on getting that part of the code to be compatible with the zipping process. Much slower going than I would have hoped...

**Comment by brichwin** [comment]
Good luck!

I made a few attempts so far to switch my web based conversion tool from MathJax to MathCAT, but realized I need to step back and try building up simpler code projects until I understand what Trunk and the web assembly process is doing. I get stymied by having very little experience debugging such a complex environment and get blocked when I get errors like:

Uncaught RuntimeError: unreachable executed
    set_lang http://localhost:8080/wasm_demo.js:120
    run http://localhost:8080/:32
wasm_demo_bg.wasm:134952:1

So, I've stepped back and trying to learn the fundamentals by building up a rust library a piece at a time with layers of exception handling to learn how web assemblies work with bundled assets.
